cloud/amazon/iam_policy.py

```diff
@@ -283,7 +283,7 @@ def main():
       iam_name=dict(default=None, required=False),
       policy_name=dict(default=None, required=True),
       policy_document=dict(default=None, required=False),
-      policy_json=dict(default=None, required=False),
+      policy_json=dict(type='dict', default=None, required=False),
```

Comment:
Also doesn't work if I wrap spaces around it, or use `to_json` or use `convert_data=False`.  Only thing that works is `type='dict'`